src/main/resources/edu/cornell/kfs/sec/cu-spring-sec.xml

```diff
@@ -1,33 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-    <!--
-        Copyright 2005-2007 The Kuali Foundation. Licensed under the
-        Educational Community License, Version 2.0 (the "License"); you may
-        not use this file except in compliance with the License. You may
-        obtain a copy of the License at
-        http://www.opensource.org/licenses/ecl2.php Unless required by
-        applicable law or agreed to in writing, software distributed under the
-        License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
-        CONDITIONS OF ANY KIND, either express or implied. See the License for
-        the specific language governing permissions and limitations under the
-        License.
-    -->
-<beans xmlns="http://www.springframework.org/schema/beans"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xmlns:p="http://www.springframework.org/schema/p"
-    xsi:schemaLocation="http://www.springframework.org/schema/beans
-        http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">    
-
-    <!--
-        The Access Security module configuration bean hierarchy has two levels of parent beans.
-        We override the one higher up in the path, to minimize required changes and to prevent
-        reloading of the whole SEC module (which could mess up our other modules' bean overrides).
-     -->
-    <bean id="secModule-parentBean" parent="secModule-coreOnly-parentBean" abstract="true">
```

Comment:
If CuSecurityModelMaintainableImpl is being kept, then this bean needs to be kept as well.

Reply:
restored file